Vagrantfile

```diff
@@ -110,6 +110,10 @@ ENV["LC_ALL"] = "en_US.UTF-8"
 ENV["LC_CTYPE"] = "en_US.UTF-8"
 
 Vagrant.configure(2) do |config|
+    if ENV['CILIUM_TEMP'] then
```

Comment:
Maybe it could have been by default instead?

Reply:
We error out if `ENV['CILIUM_SCRIPT']` is not set and set both env variables in `contrib/vagrant/start.sh`, so maybe we don't need to check `ENV['CILIUM_TEMP']`? It's always supposed to be set.
https://github.com/cilium/cilium/blob/1bf235a4cecbc844740e949165607cd25b33e1a4/Vagrantfile#L20-L26

Reply:
Ah sorry, I thought you add this new variable. LGTM the way it is.